source/js/buyers-guide/components/creepometer/creepometer.jsx

```diff
@@ -10,8 +10,7 @@ export default class Creepometer extends React.Component {
 
     this.state = {
       dragging: false,
-      percentage: 50,
-      value: 50
```

Comment:
I don't remember why `value` was there, but it literally did nothing, it was always the same as `percentage`.